src/main.cpp

```diff
@@ -2752,7 +2752,7 @@ bool static DisconnectTip(CValidationState& state, const CChainParams& chainpara
             list<CTransaction> removed;
             CValidationState stateDummy;
             if (tx.IsCoinBase() || !AcceptToMemoryPool(mempool, stateDummy, tx, false, NULL, true)) {
-                mempool.removeRecursive(tx, removed);
+              mempool.removeRecursive(tx, removed, MemPoolRemovalReason::REORG);
```

Comment:
trivial nit: indentation symmetry between `mempool.removeRecursive` line and `vHashUpdate.push_back` line.